ts/nni_manager/test/core/nnimanager.test.ts

```diff
@@ -135,7 +137,10 @@ describe('Unit test for nnimanager', function () {
 
     after(async () => {
         // FIXME
```

Comment:
What is this FIXME

Reply:
It is not a full clean up. A full clean up will kill mocha.